<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>719</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>223</red>
        <green>232</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>223</red>
        <green>232</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>223</red>
        <green>232</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>223</red>
        <green>232</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Подробные вычисления</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="results">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>311</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>311</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Нажмите на кнопку &quot;Открыть график&quot;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="open_graph_btn">
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>141</width>
       <height>28</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Открыть график</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
